### PR TITLE
feat: migrating kube-binaries to components.json from manifest.json

### PR DESCRIPTION
**Pull request by Devinwong**
**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
feat: migrating kube-binaries to components.json from manifest.json

Test VHD build and AB E2E tests passed.
https://msazure.visualstudio.com/CloudNativeCompute/_build/results?buildId=98351661&view=logs&j=df35120d-5495-5025-96f7-85e96dd85c66

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:

```
none
```

### CODE REVIEW

**Comment by jason1028kr** [comment]
Should we remove corresponding entries from manifest.json?

**Comment by Devinwong** [comment]
> Should we remove corresponding entries from manifest.json?

oh I thought I did. Let me do it now.

File: vhdbuilder/packer/test/linux-vhd-content-test.sh [hunk 1]
```diff
@@ -90,8 +90,14 @@ testPackagesInstalled() {
     returnPackageVersions ${p} ${OS} ${OS_VERSION}
     PACKAGE_DOWNLOAD_URL=""
     returnPackageDownloadURL ${p} ${OS} ${OS_VERSION}
+    if [ ${name} == "kubernetes-binaries" ]; then
+      # kubernetes-binaries, namely, kubelet and kubectl are installed in a different way so we test them separately
+      testKubeBinariesPresent $PACKAGE_VERSIONS
+      echo "---"
```
Comment (anujmaheshwari1) [inline vhdbuilder/packer/test/linux-vhd-content-test.sh:96]: nit: whats the purpose of this echo? also, for any echo, can we do echo "$test <insert echo text>"
can look at other examples of echo in tests

**Review by anujmaheshwari1 — APPROVED** [review]
approved with a minor comment

File: vhdbuilder/packer/test/linux-vhd-content-test.sh (hunk 1 above)
Comment (Devinwong) [reply vhdbuilder/packer/test/linux-vhd-content-test.sh]: Ah right. This is not needed. Removed

**Review by jason1028kr — APPROVED** [review]

**Pull request merged**